sentiment_analysis.ipynb

```diff
@@ -0,0 +1,219 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **IMPORTANT**\n",
+    "For anything in this notebook to work, you need to install the ThirdAI package by signing up [here](https://www.thirdai.com/try-bolt/)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Sentiment analysis with BOLT**\n",
+    "\n",
+    "We will walk through the entire process of building a sentiment analysis model with BOLT from data preprocessing all the way to inference. This notebook is structured as follows:\n",
+    "1. Selecting and preprocessing the dataset\n",
+    "2. Defining the BOLT network\n",
+    "3. Training the network\n",
+    "4. Inference"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### **1. Choosing and preprocessing the dataset**\n",
+    "Let's start with the Yelp Reviews dataset. As the name suggests, it's a dataset containing reviews on Yelp, each labeled with a number between 0 to 4, inclusive, representing 1 to 5 star ratings. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO: Introduce the datasets\n",
```

Comment:
Can you pare this down so it's a working example without needing the entire datasets library? Just for now to get something public and workable, MVP